### ISSUE TITLE
Looks good!

### ISSUE THREAD

**Issue by Vastrix** [pbek/movietime#1]
Hey,

First of all, can you please release binaries, I don't know which asshole started the practice (in the Linux community) to never release binaries but I wish him a utterly painful death. I mean seriously, no one likes to build programs themselves unless they made actual changes..
  
I was about to start of the tedious process of installing ruby along with the required dependencies (to then probably eventually be unable to compile the lot) when I saw that the ruby version required was pretty old. As are the latest commits...
  
  
**TL:DR**
- Can you please (always) upload binaries?
- I assume the project is finished? :o

**Comment by pbek** [comment]
First, please watch your language.
Second, there are no binaries. This project uses ruby on rails as scripting framework.
